apisix/init.lua

```diff
@@ -72,6 +72,12 @@ function _M.http_init(args)
     math.randomseed(seed)
     parse_args(args)
     core.id.init()
+
```

Comment:
why not use `logrotate`? which is supported by system

Reply:
In a container environment, users need to configure `logrotate` and `crontab`, and `alpine` image needs to install `logrotate` separately, which feels a bit complicated. 
I think it's ok to provide an optional plugin.